static/openapi/p3-spec.json

```diff
@@ -0,0 +1,3479 @@
+                      ],
+                      "purgeCdnCache": true,
+                      "cdnData": {
+                        "network": "production",
+                        "objectsType": "cpcode",
+                        "objects": [
+                          123456,
+                          456789
+                        ],
+                        "auth": {
+                          "accessToken": "ACCESS_TOKEN",
+                          "clientSecret": "CLIENT_SECRET",
+                          "clientToken": "CLIENT_TOKEN",
+                          "baseUri": "HOST"
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
+              "description": "Cache purge completed successfully.",
+              "content": {
+                "application/json": {
+                  "schema": {
+                    "type": "object",
+                    "properties": {
+                      "message": {
+                        "type": "string"
+                      },
+                      "proxyCache": {
+                        "type": "object",
+                        "properties": {
+                          "status": {
+                            "type": "object",
+                            "additionalProperties": {
+                              "type": "string"
+                            }
+                          }
+                        }
+                      }
+                    }
+                  },
+                  "examples": {
+                    "purgeResponse": {
+                      "summary": "Example response for a successful purge",
+                      "value": {
+                        "message": "Proxy cache purge completed.",
+                        "proxyCache": {
+                          "status": {
+                            "https://example.com/*": "PURGED",
+                            "https://example.com/home/*": "PURGED"
+                          }
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            },
+            "400": {
+              "description": "Bad Request."
+            },
+            "401": {
+              "description": "Authorization failure due to invalid authentication credentials."
+            },
+            "403": {
+              "description": "Either the API key doesn't have permissions or it is deactivated."
+            },
+            "500": {
+              "description": "Internal server error."
+            }
+          }
+        }
+      },
+      "/api/p3/v1/ppm/jobs": {
+        "get": {
+          "tags": [
+            "PPM"
+          ],
+          "summary": "Get list of jobs",
+          "description": "Retrieves a paginated list of PPM jobs.",
```

Comment:
Do you need to say "paginated" list?  SInce offset & limit are optional, pagination is optional.  Then it is not returning a "paginated" list.